README.md

```diff
@@ -58,122 +52,20 @@ For Example:
-                        ".........",                                  
-                        "3.40E-02"                                    
-                ],                                                    
-                "SigmaCorr": [                                        
-                        "........."                                   
-                ]                                                     
-        },                                                            
-        "ParameterStructures": {                                      
-                "Theta": 5,                                           
-                "Omega": [                                            
-                        1,                                            
-                        0,                                            
-                        1                                             
-                ],                                                    
-                "Sigma": [                                            
-                        1                                             
-                ]                                                     
-        },                                                            
-        "ParameterNames": {                                           
-                "Theta": [                                            
-                        "KA",                                         
-                        "CL",                                         
-                        "V2",                                         
-                        "RUVp",                                       
-                        "RUVa"                                        
-                ],                                                    
-                "Omega": [],                                          
-                "Sigma": []                                           
-        },                                                            
-        "OFV": 2643.561                                               
-}    
+
 ```
 
 or human readable tabular formats such as:
 
 ```
-+-----------+----------+-------------------+--------------+---------------------+
-|   Theta   |   Name   |   Estimate (SN)   |   Estimate   |      StdErr (RSE)   |
-+-----------+----------+-------------------+--------------+---------------------+
-|   TH 1    |     KA   |        2.31E+00   |       2.31   |    0.0862 (3.7 %)   |
-|   TH 2    |     CL   |        5.50E+01   |         55   |      3.33 (6.1 %)   |
-|   TH 3    |     V2   |        4.65E+02   |        465   |      29.7 (6.4 %)   |
-|   TH 4    |   RUVp   |       -8.06E-02   |    -0.0806   |   0.0555 (68.9 %)   |
-|   TH 5    |   RUVa   |        4.13E+00   |       4.13   |     1.36 (32.9 %)   |
-+-----------+----------+-------------------+--------------+---------------------+
+
 ```
 
 
```

Comment:
Is the section below still correct? I don't think so, but I could be wrong. If it's not, let's just remove it for now.